Isodose/Logic/vtkSlicerIsodoseModuleLogic.cxx

```diff
@@ -296,15 +293,24 @@ vtkMRMLColorTableNode* vtkSlicerIsodoseModuleLogic::GetDefaultIsodoseColorTable(
       vtkWarningWithObjectMacro(scene, "GetDefaultIsodoseColorTable: Multiple default isodose color table nodes found");
     }
 
-    vtkMRMLColorTableNode* isodoseColorTableNode = vtkMRMLColorTableNode::SafeDownCast(defaultIsodoseColorTableNodes->GetItemAsObject(0));
-    return isodoseColorTableNode;
+    vtkObject* object = defaultIsodoseColorTableNodes->GetItemAsObject(0);
+    if (object)

```

Comment:
I think if number of items is greater than 0 then the 0th object will always be a valid vtkObject. The only thing we need to check is its type. However, if the type is not the one you want to SafeDownCast to, then the `isodoseColorTableNode` pointer will be null, so it will return null. I personally would remove this particular check in order to keep the file shorter and thus more readable.

Reply:
I've restored origin part.